Code/Editor/AzAssetBrowser/AzAssetBrowserRequestHandler.cpp

```diff
@@ -372,6 +373,77 @@ void AzAssetBrowserRequestHandler::AddContextMenuActions(QWidget* caller, QMenu*
             AzToolsFramework::Layers::EditorLayerComponent::CreateLayerAssetContextMenu(menu, fullFilePath, levelPath);
         }
 
+        // Create the callback to pass to the SourceControlAPI
+        AzToolsFramework::SourceControlResponseCallback callback =
+            [fullFilePath](bool success, const AzToolsFramework::SourceControlFileInfo& info)
+        {
+            if (success)
+            {
+                QMessageBox::information(AzToolsFramework::GetActiveWindow(), "Asset deleted", "Asset name %s", fullFilePath.c_str());
+            }
+            else
+            {
+                // Be more specific with errors so as to give the user the best chance at fixing them
+                if (!info.HasFlag(AzToolsFramework::SCF_OpenByUser))
+                {
+                    if (info.HasFlag(AzToolsFramework::SourceControlFlags::SCF_OutOfDate))
+                    {
+                        QMessageBox::information(
+                            AzToolsFramework::GetActiveWindow(), tr("Asset not deleted"),
+                            QString(tr("Source Control Issue - You do not have latest changes from source control for file\n%1"))
```

Comment:
Nitpick: `tr` returns a `QString`, so you shouldn't need to pass it into another `QString` constructor here (and same below)